.ghjk/lock.json

```diff
@@ -856,25 +855,6 @@
           },
           "packageName": "node-gyp"
         },
-        "88f6aa5ddb513712623d31a17096ea26286064da": {
-          "version": "v2.4.0",
-          "port": {
-            "ty": "denoWorker@v1",
```

Comment:
@Yohe-Am you have a plan or strategy to stabilize the lock file and avoid change when the arch is different?